docs/sources/shared/datasources/tempo-search-traceql.md

```diff
@@ -0,0 +1,97 @@
+| 3      | Choose filter      | Choose whether to filter using **Resource Service Name**, **Span Name**, and/or **Duration**.                                                                              | Optional. You can execute an empty query in the Search tab. In TraceQL, `{}` is a valid query and is the default query until you choose options. |
+| 4      | Filters conditions | Select options for one or more filters. For example, you can define a filter where **Resource Service Name** (`resource.service.name`) equals (`=`) `cloud-logs-archiver`. | Optional. At least one tag or filter must be defined.                                                                                            |
+| 5      | Tags               | Add tags for span, resource, or unscoped and define their conditions.                                                                                                      | Optional.                                                                                                                                        |
+| 6      | TraceQL query      | Displays the TraceQL query constructed by your selections.                                                                                                                 | This TraceQL query is executed when you select **Run query**.                                                                                    |
+
+Every query searches the data for the selected time frame.
+By default, queries run against data from the last hour.
+Select Time range to the left of Run query to choose the time range for the data your query runs against.
+Read the [dashboard time range](/docs/grafana/latest/dashboards/use-dashboards/#set-dashboard-time-range) documentation to learn more.
+
+To access Search, use the following steps:
+
+1. Sign into Grafana.
+1. Select your Tempo data source.
+1. From the menu, choose Explore and select Query type > Search.
+
+## Define filters
+
+Using filters, you refine the data returned from the query by selecting Resource Service Name, Span Name, or Duration. The **Duration** represents span time, calculated by subtracting the end time from the start time of the span.
+
+Grafana administrators can change the default filters using the Tempo data source configuration.
+Filters can be limited by the operators. The available operators are determined by the field type.
+For example, **Span Name** and **Resource Service Name** are string fields so the comparison operators are equals (`=`), not equal (`!=`), or regular expressions (`=~`).
+
+**Duration** is a duration field type and uses range selections (`>`, `>=`, `<`, `<=`). When you select multiple values for the same tag, Grafana automatically changes the operator to the regex operator `=~` and concatenates the values with a `|`.
```

Comment:
```suggestion
When you select multiple values for the same tag, Grafana automatically changes the operator to the regex operator `=~` and concatenates the values with a `|`.

**Duration** is a duration field type and uses range selections (`>`, `>=`, `<`, `<=`). 
```

Reply:
Line 67 should be in a code block. The formatting in the suggestion box doesn't like the additional code block. I'll fix it once we get the suggestion wording right.

Reply:
@adrapereira How does this update look?